flink-cdc-connect/flink-cdc-source-connectors/flink-connector-mysql-cdc/src/main/java/io/debezium/connector/mysql/MySqlDefaultValueConverter.java

```diff
@@ -35,7 +35,7 @@
 import java.util.regex.Pattern;
 
 /**
- * Copied from Debezium project(1.9.7.final) to add BIGINT and SMALLINT to TRIM_DATA_TYPES. Remove
+ * Copied from Debezium project(1.9.8.final) to add BIGINT and SMALLINT to TRIM_DATA_TYPES. Remove
  * this when https://issues.redhat.com/browse/DBZ-6824 is fixed.
```

Comment:
FYI As far I can see from the issue's comments, this issue is closed since Debezium 2.3

Reply:
Thanks a lot, it seems it is fixed in [2.3.3.Final](https://issues.redhat.com/browse/DBZ-6824). Add this to the JavaDocs,